onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2547,6 +2565,7 @@ static constexpr OrtApi ort_api_1_to_12 = {
     &OrtApis::ReleaseKernelInfo,
     // End of Version 12 - DO NOT MODIFY ABOVE (see above text for more information)
 
+    &OrtApis::GetTrainingApi,
```

Comment:
As this is adding a new version 13 to the API, you need to bump this to 13 in the onnxruntime_c_api.h:

#define ORT_API_VERSION 12

And place a comment here above your line saying:
// Start of Version 13 API in progress, safe modify/rename/rearrange until we ship 

(this should have been there but I don't see it)